checksum.xml

```diff
@@ -35,6 +35,7 @@
     <trusted-key id='353a436e043e3145' group='com.google.code.findbugs' />
     <trusted-key id='59a252fb1199d873' group='com.google.code.findbugs' />
     <trusted-key id='7a01b0f236e5430f' group='com.google.code.gson' />
+    <trusted-key id='b0f3710fa64900e7' group='com.google.code.gson' />
```

Comment:
In case you wonder, the same key has already been listed for other Google groups.